core/templates/pages/topic-editor-page/topic-editor-page.component.html

```diff
@@ -129,6 +129,7 @@
     font-size: 12px;
   }
   .oppia-editor-warnings-box {
+    background: #fcf8e3;
```

Comment:
Earlier the color was added through `.uib-dropdown-menu.oppia-editor-warnings-box` ([ref](https://github.com/oppia/oppia/blob/e01116bf87df729fcc26169f612643a75d2dec3c/core/templates/css/oppia.css#L2564))